coffee-table-groups/coffee-table-groups.md

```diff
@@ -1,113 +1,116 @@
+<!-- markdownlint-disable MD024 -->
 # Coffee Table Groups
 
 > [!IMPORTANT]
-> These are the currently scheduled times for the events at the time of this publication. Please check the official Virtual Coffee `#announcements`, `#vc-events`, or other noted channels on Slack for any updates and links to event rooms.
+> These are the currently scheduled times and information for the events at the point of publication. Please check our [online events page](https://virtualcoffee.io/events) as well as the `#announcements` and `#vc-events` channels on Virtual Coffee's Slack for the latest updates and links to event rooms.
 
 ---
 
 ## Active Groups
 
 > [!NOTE]
-> We're looking for Co-hosts for:
+> We're continually accepting volunteer applications for Co-hosts for current Coffee Table Groups and Leads for new Coffee Table Groups.
 >
-> - Tech Interview Study Group
-> - DSA Office Hours
-> - Frontend Friday Folks Fighting CSSBattle.dev
->
-> Please reach out to Meg or Abbey to inquire about getting involved with either of these groups.
+> Please reach out to Meg or Abbey if you have any questions.
 
 ### Tech Interview Study Group
 
-**When:**
+#### When:
 
-- Mondays at 4:00 PM ET
+- Mondays, 4:00–5:00 PM ET
 
-**Lead & Host:**
+#### Lead & Host:
 
 - Eddie Banner
 
-A weekly discussion of job search & interview-related topics—mini mock interviews, technical topic discussions, job search tips, occasional guest speakers, etc. We are always looking for ideas to help meet your needs! All stages of career & job search are welcome.
+A weekly discussion of job search and interview-related topics, including mini mock interviews, technical topic discussions, job search tips, occasional guest speakers, and more! We are always looking for ideas to help meet your needs. All stages of career and job search welcome!
 
 Interested? Join the `#tech-interview-study-group` Slack channel!
 
 ### Accountabilibuddies
 
-**When:**
+#### When:
 
-- Tuesdays at 7:00–9:00 PM ET
-- Thursdays at 9:00–11:40 AM ET
+- **The Quad** – Tuesdays, 7:00–9:00 PM ET
```

Comment:
**Question**: As a non-native English, I can get "breakfast club" means. But what does "quad" mean?

Reply:
"Quad" is a prefix meaning four (think quadruple). @radturkin suggested it because he and @david-001 are going with a new format for their Accountabilibuddies session where they're breaking it up into four Pomodoro sessions. I will let them share their thoughts here as well. They may be able to come up with another title.